# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmBuiltins.h
# Commit: 67ae30727ed8
# Full Hash: 67ae30727ed835cf73153a4ee8f0c9c7931aab09
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2021-12-23 09:34:36
# Regressor Bug: 1642412
# File Overlap Count: 7
# Description:
#   Bug 1642412 - wasm: Remove unnecessary code from Instance for inline table methods. r=lth
#   
#   This commit removes support for anyref in Instance::tableGet/tableSet,
#   making it clear that these functions are only used for function tables.
#   Instance::tableSize is removed as no longer needed.
# ==============================================================================

diff -r 90fa382b5a20 -r 67ae30727ed8 js/src/wasm/WasmBuiltins.h
--- a/js/src/wasm/WasmBuiltins.h	Wed Dec 22 17:36:30 2021 +0000
+++ b/js/src/wasm/WasmBuiltins.h	Wed Dec 22 17:36:30 2021 +0000
@@ -111,11 +111,10 @@
   TableCopy,
   ElemDrop,
   TableFill,
-  TableGet,
+  TableGetFunc,
   TableGrow,
   TableInit,
-  TableSet,
-  TableSize,
+  TableSetFunc,
   RefFunc,
   RefTest,
   RttSub,
@@ -237,11 +236,10 @@
 extern const SymbolicAddressSignature SASigTableCopy;
 extern const SymbolicAddressSignature SASigElemDrop;
 extern const SymbolicAddressSignature SASigTableFill;
-extern const SymbolicAddressSignature SASigTableGet;
+extern const SymbolicAddressSignature SASigTableGetFunc;
 extern const SymbolicAddressSignature SASigTableGrow;
 extern const SymbolicAddressSignature SASigTableInit;
-extern const SymbolicAddressSignature SASigTableSet;
-extern const SymbolicAddressSignature SASigTableSize;
+extern const SymbolicAddressSignature SASigTableSetFunc;
 extern const SymbolicAddressSignature SASigRefFunc;
 extern const SymbolicAddressSignature SASigPreBarrierFiltering;
 extern const SymbolicAddressSignature SASigPostBarrier;